.github/workflows/test.yml

```diff
@@ -78,6 +78,20 @@ jobs:
           cd bindings/python
           pytest -vs tests
 
+      - name: Update to dev version and build release
+        if: github.ref == 'refs/heads/main' and python-version: 3.6
+        run: |
+          cd ../../build/bindings/python
+          pip install setuptools wheel twine build
+          sed "s/0.0.1/$(openssl rand -base64 12)/" setup.py
```

Comment:
Can you prefix this with dev like: sed "s/0.0.1/0.0.1-dev-$(openssl rand -base64 12)/" setup.py

Reply:
Couldn't reply to previous comment as there's no chat box. 

install_genalyzer.sh is used here - https://github.com/analogdevicesinc/genalyzer/blob/d941da9bc3d124e73abf18e6e9dc1002713e762b/.github/workflows/test.yml#L71
It's not a blank file.